README.md

```diff
@@ -82,27 +89,20 @@ Copy all folders in **Relativ/src/libraries** and past them in your Arduino Libr
 
 # Building the hardware
 
-Now what you need to do is 3D-print the hardware; if needed you can change any models with provided Source Files. 
-You will need to print the following :
+**A detailed documentation can be found on our <a href="https://wiki.relativty.net/index.php/Main_Page">wiki<a/>!**
+  
+Now what you need to do is 3D-print the hardware; if needed you can change any models with provided Source Files.
 
-* (1x): Part 1, Part 2a, Part 2b, Part 3, Part 3a, Part 4, Part 5
-* (2x): Part 2c, Part 2d, Part 3b
-* (1x) Part 6: Use of Flexible filament with 10% to 50% infill density
-* (3x) Part 7: Use of Flexible filament with 10% to 50% infill density
-* (1x) Part 8: Use of Flexible filament with 5% infill density (it makes it like a sponge)
-* If you uses the the short straps (190mm, because the large ones don't fit in your printer for example), there are extensions (part 9) that should be printed aswell (2x).
-
-⚠️ **PAY EXTRA ATTENTION TO THE SCREEN, IT'S VERY FRAGILE** ⚠️
-
-Additional build information can be found in the build manuals in the 3D-Models folders.
+⚠️ PAY EXTRA ATTENTION TO THE SCREEN, IT'S VERY FRAGILE ⚠️
```

Comment:
Becouse there are different kinds of 3D models, this information was kinda outdatet, so I have removed it.